aptos-move/framework/aptos-framework/sources/storage_gas.move

```diff
@@ -393,17 +393,20 @@ module aptos_framework::storage_gas {
             error::already_exists(ESTORAGE_GAS_CONFIG)
         );
 
+        let k: u64 = 1000;
+        let m: u64 = 1000 * 1000;
+
         let item_config = UsageGasConfig {
             target_usage: 1000000000, // 1 billion
-            read_curve: base_8192_exponential_curve(80000, 80000 * 100),
-            create_curve: base_8192_exponential_curve(2000000, 2000000 * 100),
-            write_curve: base_8192_exponential_curve(400000, 400000 * 100),
+            read_curve: base_8192_exponential_curve(5 * m, 5 * m * 100),
```

Comment:
Drive by comment: I'm surprised you are using unchecked arithmetics here. Are you sure `f * m * 100` can never overflow? Can `m` depend on input? I thought that all our gas related arithmetics uses `checked_<op>` or `saturated_<op>`.

Reply:
> Drive by comment: I'm surprised you are using unchecked arithmetics here. Are you sure `f * m * 100` can never overflow? Can `m` depend on input? I thought that all our gas related arithmetics uses `checked_<op>` or `saturated_<op>`.

Alright seeing now `m` is a constant.